myhpi/static/js/myHPI.js

```diff
@@ -70,6 +70,7 @@ const localizeLastPublished = () => {
         let abbr = document.createElement("abbr");
         abbr.setAttribute("title", lastPublishedLocalized)
         lastPublished.removeAttribute("title")
+        lastPublished.innerHTML = lastPublished.textContent.trim()
```

Comment:
why is this change necessary?

Reply:
Does this also work with innerText to get rid of this security warning?